chore: update Kotlin/Wasm and Compose Multiplatform tutorial (#5541)
JetBrains/kotlin-web-site: docs/topics/wasm/wasm-get-started.md

<primary-label ref="beta"/> 

This tutorial demonstrates how to run a [Compose Multiplatform](https://www.jetbrains.com/lp/compose-multiplatform/) 
app with [](wasm-overview.md) in IntelliJ IDEA and generate artifacts you can publish as a website.

## Create a project

1. [Set up your environment for Kotlin Multiplatform development](https://kotlinlang.org/docs/multiplatform/quickstart.html#set-up-the-environment).
2. In IntelliJ IDEA, select **File | New | Project**.
3. In the list of project templates, select **Kotlin Multiplatform**.

   > If you're not using the Kotlin Multiplatform IDE plugin, you can generate the same project using the [KMP web wizard](https://kmp.jetbrains.com/?web=true&webui=compose&includeTests=true).
   >
   {style="tip"}

4. Specify the following fields in the **New Project** window:

   * **Name:** WasmDemo
   * **Project ID:** wasm.project.demo

   > This tutorial uses `wasm.project.demo` as the Project ID for consistency. 

>    > However, we recommend keeping your usual Project ID, such as `org.example`.

Reviewer: ```suggestion
   > However, we recommend keeping your usual group ID, such as `org.example`.
```

## Run the application

1. Once the project loads, select **webApp [wasmJs]** in the list of run configurations and click **Run**.

    ![Run the Compose Multiplatform app on web](compose-run-web-light.png){width=300}
    
    The web application opens automatically in your browser. 
    Alternatively, once the build finishes, you can open the following URL manually:
    
    ```shell
       http://localhost:8080/
    ```
    
    The port number may vary if port `8080` is already in use.
    You can find the actual port number in the output of the Gradle build.

> 2. Click the **Click me!** button:

Reviewer: I'd avoid lists that consist of two huge (onscreen) items — it may be hard to see why the formatting is off, and there is no real sequence here as the only non-obvious action is the run configuration.

On the same note, it's probably worth it to get rid of the second screenshot in the "click me" section, how many screenshots are really needed for the user to click the "Click me" button. The end result should be sufficient.

    ./gradlew wasmJsBrowserDistribution
    ```

Once the task completes, you can find the generated artifacts in the `webApp/build/dist/wasmJs/productionExecutable`
directory:

![Artifacts directory](wasm-composeapp-directory.png){width=400}

## Publish the application

Use the generated artifacts to deploy your Kotlin/Wasm application. 

> Select a publishing option of your preference and follow the instructions:

Reviewer: ```suggestion
Select a publishing option that you prefer and follow the instructions:
```
